src/renderer/components/Commit.vue

```diff
@@ -0,0 +1,466 @@
+        <div class="since-commit">
+          <div class="icons">
+            <aph-icon name="hex"></aph-icon>
+            <aph-icon name="star"></aph-icon>
+          </div>
+          <div class="exchange-val">
+            <div class="label">{{$t('totalFeesCollectedSinceCommit')}}</div>
+            <div class="highlight">{{$formatNumber($store.state.commitState.feesCollectedSinceCommit)}} APH</div>
+            <div class="estimate"><span>{{ $t('estimated') }} ({{ $store.state.currency }})</span> {{$formatNumber(aphHolding ? aphHolding.unitValue * $store.state.commitState.feesCollectedSinceCommit : 0)}}</div>
+          </div>
+        </div>
+      </div>
+      <div class="user-values">
+        <div class="value">
+          <h2 class="underlined">{{$t('amountCommitted')}}</h2>
+          <div class="highlight">{{$formatNumber($store.state.commitState.quantityCommitted)}} APH</div>
+          <div class="lower"><span>{{ $t('estimated') }} ({{ $store.state.currency }})</span> {{$formatNumber(aphHolding ? aphHolding.unitValue * $store.state.commitState.quantityCommitted : 0)}}</div>
+        </div>
+        <div class="date">
+          <h2 class="underlined">{{$t('dateCommitted')}}</h2>
+          <div class="highlight">{{$formatDate($store.state.commitState.contributionTimestamp)}}</div>
+          <div class="lower">
+            <span>{{$t('time')}}</span>{{$formatTime($store.state.commitState.contributionTimestamp)}}
+            <span>{{$t('block')}}</span>{{$store.state.commitState.contributionHeight}}
+          </div>
+        </div>
+        <div class="weight">
+          <h2 class="underlined">{{$t('myWeight')}}</h2>
+          <div class="highlight">{{$formatNumber($store.state.commitState.weightPercentage)}}%</div>
+          <div class="lower"><span>{{$t('myWeight')}}</span>{{$formatNumber($store.state.commitState.userWeight)}}</div>
+          <div class="lower"><span>{{$t('networkWeight')}}</span>{{$formatNumber($store.state.commitState.networkWeight)}}</div>
+        </div>
+        <div class="earned">
+          <h2 class="underlined">{{$t('feesEarned')}}</h2>
+          <div class="highlight">~{{$store.state.commitState.availableToClaim}} APH</div>
+          <div class="lower">{{ $t('estimated') }} ({{ $store.state.currency }}) {{$formatNumber(aphHolding ? aphHolding.unitValue * $store.state.commitState.availableToClaim : 0)}}</div>
+        </div>
+      </div>
+      <div class="actions">
+        <div class="commit">
+          <div class="btn-square" @click="showCommitModal()">
+            <aph-icon name="commit"></aph-icon>
+            <p>{{$t('commit')}}</p>
+          </div>
+        </div>
+        <div class="claim-info">
+          <div class="value" v-if="$store.state.commitState.quantityCommitted > 0">
+            <h2>{{$store.state.commitState.ableToCompoundHeight - currentBlock}}</h2>
+            <div>{{$t('blocksUntilCompound')}} ({{$store.state.commitState.ableToCompoundHeight}})</div>
```

Comment:
This value will go negative once more than the ableToCompoundHeight has been passed. Should just not show the value or show 0.